packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md

```diff
@@ -357,8 +357,8 @@ class Animal {
   numLegs: number = 4;
 }
 
-class Bee extends Animal {
```

Comment:
Not to be a buzzkill (pun intended), but, I'm not super sure if this is the right change; reasonably the "fix" should have been to override `numLegs` with `numLegs = 6` in the `Bee` class, right? And that's why it was declared as `number` in `Animal`?

Also if we want to be animal pendants... cats shouldn't drink milk! 🐱